.github/workflows/validate_urls.yml

```diff
@@ -0,0 +1,39 @@
+name: Validate URLs
+
+on:
+  page_build:
+  schedule:
+    - cron: '00 14,23 * * *'
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to validate'
```

Comment:
Can we make this description more useful?

Reply:
We can.  This is the name of the field when you manually dispatch the workflow.  Do you have any suggestions?

Reply:
![Screen Shot 2021-11-08 at 5 13 21 PM](https://user-images.githubusercontent.com/79471462/140838254-7b72aaa3-2d08-46e2-86f8-9233f2ca1abd.png)
Here is an example of what it would look like.

Reply:
Looks like this would *not* be for changing the branch from gh-pages to something else. 

"Name of branch to link crawl"

"Branch to crawl for broken links"

"Branch containing the markdown and swift you wish to crawl"

"Branch me, bro!"

Something like that (maybe not the last one)

Reply:
How 'bout `Branch to crawl for broken links in .md and .swift files`

Reply:
Honestly, despite the length, I appreciate the context. I like it better.